clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/security/GoogleNamedAccountCredentials.java

```diff
@@ -99,6 +99,10 @@ public String regionFromZone(String zone) {
       return null;
     }
 
+    public List<String> getZonesFromRegion(String region) {
+      return region != null && regionToZonesMap != null ? regionToZonesMap.get(region) : null;
```

Comment:
You don't need the `region != null`, because there is no `null` key in the map anyway.